db/postgres.go

```diff
@@ -221,6 +224,51 @@ func (p *pgClient) WriteTodaysUsage(counts map[string]api.RelayCounts) error {
 	return nil
 }
 
+// WriteTodaysUsage writes the app metrics for today so far to the underlying PG table.
+//	All the entries in the table holding todays metrics are deleted first.
+func (p *pgClient) WriteTodaysLatencies(latencies map[string][]api.Latency) error {
+	ctx := context.Background()
+	// TODO: determine required isolation level
+	tx, err := p.DB.BeginTx(ctx, &sql.TxOptions{Isolation: sql.LevelSerializable})
+	if err != nil {
+		return err
+	}
+
+	// todays_app_latencies table gets rebuilt every time
+	_, deleteErr := tx.ExecContext(ctx, "DELETE FROM todays_app_latencies")
+	if deleteErr != nil {
+		if rollbackErr := tx.Rollback(); rollbackErr != nil {
+			fmt.Printf("delete failed: %v, unable to rollback: %v\n", deleteErr, rollbackErr)
+			return deleteErr
+		}
+	}
+
+	// TODO: bulk insert
+	for app, latencies := range latencies {
```

Comment:
This variable naming works, but is confusing due to using the same name twice.

Reply:
Fixed